build.sbt

```diff
@@ -510,9 +532,8 @@ lazy val scioExamples: Project = Project(
   "scio-examples",
   file("scio-examples")
 ).settings(
-  commonSettings ++ noPublishSettings ++ soccoSettings,
+  commonSettings ++ noPublishSettings ++ soccoSettings ++ beamRunnerSettings,
   libraryDependencies ++= Seq(
-    "org.apache.beam" % "beam-runners-direct-java" % beamVersion,
```

Comment:
We should either keep this or change `def beamRunnerSettings` to include `DirectRunner` by default?

Reply:
@nevillelyh `beamRunnerSettings` includes `DirectRunner` if no `-DbeamRunners` or `BEAM_RUNNERS=` is found. 

https://github.com/spotify/scio/pull/963/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR207

https://github.com/spotify/scio/pull/963/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR209

Reply:
@nevillelyh the idea is that ... if we need a runner in some subproject we just include beamRunnerSettings

Reply:
LOL missed `.getOrElse(Seq(directRunnerDependency))`. LGTM.